pants

```diff
@@ -47,7 +47,7 @@ source ${HERE}/build-support/bin/native/bootstrap_code.sh
 
 PANTS_EXE="${HERE}/src/python/pants/bin/pants_loader.py"
 
-PY=${PY:-$(which python2.7)}
+PY=${PY:-$(which python2.7 || exit 0)}
```

Comment:
It is also not obvious that anything uses this variable, as it seems like all consumers calculate it directly: ie, https://github.com/pantsbuild/pants/blob/master/build-support/virtualenv#L16

Reply:
...kicked off another travis run to try this out over here, as it seems to work locally: https://github.com/pantsbuild/pants/pull/6102

Reply:
Would be good to remove this if possible; sadly your CI run was polluted by the bad wheel thing :(

Reply:
IIRC this was here to support Twitter internal scripts at some distant point in time. I'd definitely kill.